lib/workers/branch/reuse.ts

```diff
@@ -58,10 +65,11 @@ export async function shouldReuseExistingBranch(
       if (await isBranchModified(branchName)) {
         // TODO: Warn here so that it appears in PR body (#9720)
         logger.debug('Cannot rebase branch as it has been modified');
-        return { reuseExistingBranch: true, isModified: true };
+        result.reuseExistingBranch = true;
+        result.isModified = true;
```

Comment:
Is the `return` missing here accidentally?

Reply:
Yup 😅

Reply:
Not exactly, though, my intent was to return couple lines after. What I missed is excessive debug statement.